js/data/pokedex.js

```diff
@@ -10579,5 +10579,75 @@ var POKEDEX_XY = $.extend(true, {}, POKEDEX_BW, {
     },
     "w": 305.0,
     "ab": "Aura Break"
+  },
+  "Zygarde-Cell": {
+    "t1": "Dragon",
+    "t2": "Ground",
+    "bs": {
+      "hp": ,
```

Comment:
You can use placeholder values like `0` or `undefined` or something, instead of invalid JavaScript.